brayns/engine/renderer/Renderer.cpp

```diff
@@ -34,6 +49,7 @@ Renderer &Renderer::operator=(const Renderer &other)
     _handle = ospray::cpp::Renderer(_handleName);
     _data = other._data->clone();
     _data->pushTo(_handle);
+    _flag = true;
```

Comment:
This looks a bit weird, I would do something like _flag.setModified(true).

Reply:
Will change on next PR